### DISCUSSION TITLE
Define bootstrap server using domain make cluster is offline

### DISCUSSION THREAD

**Discussion by majma24** [provectus/kafka-ui#4106] [Q&A] [answered]
Hi, I have a question. When I define a bootstrap server with a domain (ex: kafka-ui.internal.com), the cluster status is offline, but when using IP address broker the cluster status is online.

**Comment by majma24** [comment]
this happen , when i enable network policy

**Comment by Haarolean** [comment, marked as answer]
Your host must be reachable from within a container. Try pinging/telnetting the host to determine if it's available.
